plugins/GSdx/GSSetting.cpp

```diff
@@ -55,7 +55,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_ALPHAHACK:
 			return "Different alpha handling. Can work around some shadow problems.";
 		case IDC_OFFSETHACK:
-			return "Might fix some misaligned fog, bloom, or blend effect.";
+			return "Might fix some misaligned fog, bloom, or blend effect.\n"
+				"Special cases are only useful in couple of games like Captain Tsubasa.\n"
+				"\n Special cases are disabled from code side for now.";
```

Comment:
Maybe add some note stating that ``Normal (Vertex)`` option is recommended to be used? I know you've added ``Special cases are only useful in couple of games like Captain Tsubasa.`` which insinuates that ``Normal (Vertex)`` should be preferred but best to be explicit here.